L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -14,6 +14,8 @@
 ak4PFL1PF      = _ak4PFJets.clone(src = 'l1pfCandidates:PF')
 ak4PFL1Puppi   = _ak4PFJets.clone(src = 'l1pfCandidates:Puppi')
 
+from L1Trigger.Phase2L1Jets.L1SeedConeJetProducer_cff import L1SeedConeJetProducer
+
```

Comment:
missing SeedConeJet -> SeedConePFJet in this file, I think

Reply:
I've updated